src/game_interpreter_map.cpp

```diff
@@ -636,17 +718,33 @@ bool Game_Interpreter_Map::CommandOpenSaveMenu(lcf::rpg::EventCommand const& /*
 	return false;
 }
 
-bool Game_Interpreter_Map::CommandOpenMainMenu(lcf::rpg::EventCommand const& /* com */) { // code 11950
+bool Game_Interpreter_Map::CommandOpenMainMenu(lcf::rpg::EventCommand const& com) { // code 11950
 	auto& frame = GetFrame();
 	auto& index = frame.current_command;
 
 	if (Game_Message::IsMessageActive()) {
 		return false;
 	}
 
-	Scene::instance->SetRequestedScene(std::make_shared<Scene_Menu>());
-	++index;
-	return false;
+	int subscreen_id = -1, actor_index = 0;
+	bool is_db_actor = false;
+
+	if (com.parameters.size() > 0) {
```

Comment:
are these additional event command parameters part of the direct menu patch? In Makerpendium I only see the mention of variables 3326 and 3327.

Reply:
As stated in the initial post, this part is in addition to the original DirectMenu patch. I guess the proper way would be to add new custom command codes for anything that goes beyond known patches.

Reply:
right. shouldn't comment so late at night :sweat_smile: